include/swift/Parse/ParseVersion.h

```diff
@@ -0,0 +1,45 @@
+//===--- VersionParser.h - Parser Swift Version Numbers ---------*- C++ -*-===//
```

Comment:
This doesn't match the name of the file.  `VersionParser.h` seems more in line with the rest of the tree.